include/irr/asset/IAsset.h

```diff
@@ -272,6 +272,11 @@ class SAssetBundle
 			return {m_contents->begin(), m_contents->end()};
 		}
 
+		inline std::pair<core::smart_refctd_ptr<IAsset>*, core::smart_refctd_ptr<IAsset>*> getContents()
+		{
+			return {m_contents->begin(), m_contents->end()};
+		}
```

Comment:
should that be public for everyone, or should we have a special getter just for loaders?